### PR TITLE
add Repaintless.css v1.3.1 with autoupdate config

### LINKED ISSUE #7192: [Request] Add Repaintless.css
**Library name:** Repaintless.css
**Git repository url:** https://github.com/szynszyliszys/repaintless
**npm package url(optional):** repaintless
**Official homepage:** http://szynszyliszys.github.io/repaintless/

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7192 
@kennynaoh please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/szynszyliszys/repaintless
 * Official website (optional, not the repository): http://szynszyliszys.github.io/repaintless/
 * NPM package url (optional): https://www.npmjs.com/package/repaintless
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 458
   - Count of watchers: 22
   - Count of forks: 17
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 15
   - Downloads in the last month: 102

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7192, cc @szynszyliszys

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Please remove `gulpfile.js`, it's for gulp.

File: ajax/libs/Repaintless.css/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "url": "git+https://github.com/szynszyliszys/repaintless.git"
+  },
+  "version": "1.3.1",
+  "npmName": "repaintless",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "*.js",
+        "repaintless-css/repaintless*"
```
Comment (PeterDaveHello) [inline ajax/libs/Repaintless.css/package.json]: If you use `"repaintless-css/repaintless*"`, you should make a "`repaintless-css`" folder for css files.

**Comment by pvnr0082t** [comment]
OK, I got it.

File: ajax/libs/Repaintless.css/package.json (hunk 1 above)
Comment (pvnr0082t) [inline ajax/libs/Repaintless.css/package.json]: Okay, I will fix it. Thank you.

**Comment by pvnr0082t** [comment]
I have updated the ```package.json``` and removed ```gulpfile.js``` file.
@PeterDaveHello Could you please help me review again? Thank you.

File: ajax/libs/Repaintless.css/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "sass",
+    "scss",
+    "animations",
+    "performance",
+    "FLIP"
+  ],
+  "license": "MIT",
+  "name": "Repaintless.css",
+  "filename": "repaintless.min.css",
+  "optionalDependencies": {},
```
Comment (PeterDaveHello) [inline ajax/libs/Repaintless.css/package.json]: Please remove empty section.

File: ajax/libs/Repaintless.css/package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
+  "author": {
+    "email": "amjmigas@gmail.com",
+    "name": "Anna Migas"
+  },
+  "dependencies": {},
```
Comment (PeterDaveHello) [inline ajax/libs/Repaintless.css/package.json]: Please remove empty section.

File: ajax/libs/Repaintless.css/package.json (hunk 2 above)
Comment (pvnr0082t) [inline ajax/libs/Repaintless.css/package.json]: Okay. I fixed it. Thank you.

File: ajax/libs/Repaintless.css/package.json (hunk 3 above)
Comment (pvnr0082t) [inline ajax/libs/Repaintless.css/package.json]: Okay. I fixed it. Thank you.

**Pull request merged**